.github/workflows/commit.yaml

```diff
@@ -24,12 +24,6 @@ jobs:
   check:
     name: Pre-commit check, Go-${{ matrix.go-version }}
```

Comment:
intentionally not inheriting the "keep old" version in the pre-commit check as it effects which tooling can be used (ex linters)